content-src/lib/selectLayoutRender.js

```diff
@@ -82,44 +90,67 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
     return { ...component, data };
   };
 
-  const handleComponent = component => {
-    positions[component.type] = positions[component.type] || 0;
-
-    const feed = feeds.data[component.feed.url];
-    let data = {
-      recommendations: [],
-    };
-    if (feed && feed.data) {
-      data = {
-        ...feed.data,
-        recommendations: [...(feed.data.recommendations || [])],
-      };
+  const handleSpocs = (data, component) => {
+    if (!data) {
+      return data;
     }
-
-    if (component && component.properties && component.properties.offset) {
-      data = {
-        ...data,
-        recommendations: data.recommendations.slice(
-          component.properties.offset
-        ),
-      };
-    }
-
-    // Ensure we have recs available for this feed.
-    const hasRecs = data && data.recommendations;
-
+    let result = [...data];
     // Do we ever expect to possibly have a spoc.
     if (
-      hasRecs &&
       component.spocs &&
       component.spocs.positions &&
       component.spocs.positions.length
     ) {
+      const placement = component.placement || {};
+      const placementName = placement.name || "spocs";
+      const spocsData = spocs.data[placementName] || spocs.data.spocs;
```

Comment:
@ScottDowne Should we see spocs by placement in console here using your gist? I am seeing only spocs.data.spocs

Reply:
Yeah, if you log `placementName`, it should be topsites and spocs, and the spocs.data if you log it, should have both sets of spocs.

Try clearing cache? I can try again in a sec to confirm.

Reply:
I think this got fixed.